Xamarin.Forms.Controls/CoreGalleryPages/RadioButtonCoreGalleryPage.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Xamarin.Forms.CustomAttributes;
+
+namespace Xamarin.Forms.Controls
+{
+	class RadioButtonCoreGalleryPage : CoreGalleryPage<RadioButton>
+	{
+		protected override bool SupportsFocus => false;
+		protected override bool SupportsTapGestureRecognizer => true;
+		protected override void InitializeElement(RadioButton element)
+		{
+			element.Text = "RadioButton";
+		}
+
+		protected override void Build(StackLayout stackLayout)
+		{
+			base.Build(stackLayout);
+
+			IsEnabledStateViewContainer.View.Clicked += (sender, args) => IsEnabledStateViewContainer.TitleLabel.Text += " (Tapped)";
+
+			var borderButtonContainer = new ViewContainer<RadioButton>(Test.Button.BorderColor,
+				new RadioButton
+				{
+					Text = "BorderColor",
+					BackgroundColor = Color.Transparent,
+					BorderColor = Color.Red,
+					BorderWidth = 1,
+				}
+			);
+
+			var borderRadiusContainer = new ViewContainer<RadioButton>(Test.Button.BorderRadius,
```

Comment:
Thos should be CornerRadius no ? but it also doesn't work for me if i change to something like:

```
	var borderRadiusContainer = new ViewContainer<RadioButton>(Test.Button.BorderRadius,
				new RadioButton
				{
					Text = "BorderRadius",
					BackgroundColor = Color.Transparent,
					BorderColor = Color.Red,
					BorderWidth = 1,
					CornerRadius = 5
				}
			);
```